joblib/test/test_my_exceptions.py

```diff
@@ -2,6 +2,7 @@
 Test my automatically generate exceptions
 """
 from joblib import my_exceptions
+from _pytest.assertion.util import BuiltinAssertionError
```

Comment:
Oh okay. I thought `AssertionError` as a testcase must be existing due to some corner case.